src/api/daemon_info.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"errors"
+	"os/exec"
+	"strings"
+)
+
+const systemd = "systemctl"
+
+type Daemon struct {
+	name      string
+	isActive  string
+	isEnabled string
```

Comment:
IsActive as well as isEnabled should be boolean type

Reply:
systemctl is-active returns 5 possible options and is-enabled 3